core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -81,6 +83,8 @@
     private Endpoint endpoint;
     @Nullable
     private final String fragment;
+    @Nullable
+    private final ServiceRequestContext serviceContext;
```

Comment:
Hmm... `root` or `rootContext` like @anuraaga suggested? Feel free to ignore if you don't like it. :laughing:

Reply:
Let me use `rootContext`. It's probably better. 😄